src/Components/PetApplications.js

```diff
@@ -1,14 +1,42 @@
 import React from 'react';
 import Header from './Header';
 import ApplicationContainer from './ApplicationsContainer';
+import { useParams } from 'react-router-dom';
+import { gql, useQuery } from '@apollo/client';
 import '../Styles/PetApplications.scss';
 
+
+const GET_PET_APPLICATION = gql`
+query getPetById($id: ID!) {
+  getPetById(id: $id) 
+  {
+    name
+    applications {
+      name
+      email
+      description
+    }
+  }
+}
+`
+
 const PetApplications = () => {
+  let { id } = useParams();
+  const { loading, error, data } = useQuery(GET_PET_APPLICATION, {variables: {id}})
+
+
+  if (loading) {
+    return <p>Loading...</p>
+  }
+  if (error) {
+    return <p>{error}</p>
```

Comment:
We've set it up the error like this for all our queries. I'm thinking we'll want to redirect to our Error Page at some point.